src/canopy/llm/octoai.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional, Any
+import os
+from canopy.llm import OpenAILLM
+from canopy.llm.models import Function
+from canopy.models.data_models import Messages
+
+OCTOAI_BASE_URL = "https://text.octoai.run/v1"
+
+
+class OctoAILLM(OpenAILLM):
+    """
+    OctoAI LLM wrapper built on top of the OpenAI Python client.
+
+    Note: OctoAI requires a valid API key to use this class.
+          You can set the "OCTOAI_API_KEY" environment variable.
+    """
+
+    def __init__(
+        self,
+        model_name: str = "mistral-7b-instruct-fp16",
+        *,
+        base_url: Optional[str] = OCTOAI_BASE_URL,
+        api_key: Optional[str] = None,
+        **kwargs: Any,
+    ):
+        ae_api_key = api_key or os.environ.get("OCTOAI_API_KEY")
+        if not ae_api_key:
+            raise ValueError(
+                "OctoAI API key is required to use OctoAI. "
+                "If you haven't done it, please sign up at https://octo.ai"
+                "The key can be provided as an argument or via the OCTOAI_API_KEY environment variable."
+            )
+        ae_base_url = base_url
```

Comment:
ae_api_key and ai_base_url stands for anyscale_api_key and anyscale_base_url. Can we prefix them with octoai instead of ae?